datavirt64/dynamicvdb-datafederation/mysql/README.md

```diff
@@ -0,0 +1,33 @@
+## Overview
+
+This directory contains details specific to using this quickstart with a MySQL database.  It contains the [customer-schema.sql](./customer-schema.sql) file which includes the DDL and some seed data, along with this readme.
+
+## Initializing the Database
+
+The MySQL database can be setup by instantiating one of the MySQL templates (e.g. `mysql-ephemeral`) with the following parameters:
+
+* DATABASE\_SERVICE\_NAME=accounts-mysql5
+* MYSQL\_USER=myuser
+* MYSQL\_PASSWORD=mypass
+* MYSQL\_DATABASE=accounts
+
+The username and password parameters match what is defined in [datasources.env](../datasources.env).  If you use a different user and/or password, you must modify the settings in this file to match what you're using for the DB.  After modifying the file, you will have to recreate the secret so the application can pick up the changes.  You may also have to trigger a new deployment or recreate the pod, as changes to secrets are not propagated to running pods.
+
+Once the database is up and running, the tables and seed data can be installed by invoking the following:
+
+```
+$ cat customer-schema.sql |  oc exec accounts-mysql5-1-22hwq -i -- scl enable rh-mysql56 -- mysql -u '$MYSQL_USER' '-p$MYSQL_PASSWORD' -h '$ACCOUNTS_MYSQL5_SERVICE_HOST' accounts
```

Comment:
**'-p**$MYSQL_PASSWORD' -> **-p '**$MYSQL_PASSWORD'

Reply:
in MySQL (Ephemeral) template there is only new version /opt/rh/**rh-mysql57** and this version causes:
ERROR 1067 (42000) at line 57: Invalid default value for 'DATECLOSED'